styleguide/haml_locals.rb

```diff
@@ -243,5 +243,25 @@
   <p><a href="https =>//www.citizensadvice.org.uk/immigration/staying-in-the-uk-after-brexit/keeping-your-family-in-the-uk-after-brexit/">Check if you can apply to the EU Settlement Scheme</a></p>'
   },
 
-  'success_message' => 'Thank you for your feedback'
+  "success_message" => "Thank you for your feedback",
```

Comment:
Worth me showing the approach I've tried in https://github.com/citizensadvice/design-system/pull/242? It would be good to show both versions of the related content component and we can avoid the need to put data in this locals file.